go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/mwielbut/pointy v1.1.0
 	github.com/spf13/cast v1.5.0
 	github.com/zclconf/go-cty v1.13.1
-	go.mongodb.org/atlas v0.26.0
+	go.mongodb.org/atlas v0.26.2-0.20230518140002-aee9ff7710a0
```

Comment:
is this a pined version or has our versioning naming changed recently?

Reply:
This version point to a commit SHA on master that has the recently added resource in the go sdk. As soon as we release a new SDK version, we will update this version to the latest release version.